src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -4,7 +4,6 @@
 using System;
```

Comment:
@daxian-dbw, your last commit had 6 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Utils_GetPolicySetting_BothConfigFilesNotEmpty
```powershell
Assert.Null() Failure
Expected: (null)
Actual:   ScriptExecution { EnableScripts = True, ExecutionPolicy = "RemoteSigned" }
   at PSTests.Sequential.PowerShellPolicyFixture.CompareScriptExecution(ScriptExecution a, ScriptExecution b) in D:\a\1\s\test\xUnit\csharp\test_PSConfiguration.cs:line 135
   at PSTests.Sequential.PowerShellPolicyTests.Utils_GetPolicySetting_BothConfigFilesNotEmpty() in D:\a\1\s\test\xUnit\csharp\test_PSConfiguration.cs:line 446
```
Utils_GetPolicySetting_EmptySystemConfig
```powershell
Assert.Null() Failure
Expected: (null)
Actual:   ScriptExecution { EnableScripts = null, ExecutionPolicy = "RemoteSigned" }
   at PSTests.Sequential.PowerShellPolicyFixture.CompareScriptExecution(ScriptExecution a, ScriptExecution b) in D:\a\1\s\test\xUnit\csharp\test_PSConfiguration.cs:line 135
   at PSTests.Sequential.PowerShellPolicyTests.Utils_GetPolicySetting_EmptySystemConfig() in D:\a\1\s\test\xUnit\csharp\test_PSConfiguration.cs:line 646
```
PowerShellConfig_GetPowerShellPolicies_EmptySystemConfig
```powershell
Assert.NotNull() Failure
   at PSTests.Sequential.PowerShellPolicyTests.PowerShellConfig_GetPowerShellPolicies_EmptySystemConfig() in D:\a\1\s\test\xUnit\csharp\test_PSConfiguration.cs:line 412
```
PowerShellConfig_GetPowerShellPolicies_BothConfigFilesNotEmpty
```powershell
Assert.NotNull() Failure
   at PSTests.Sequential.PowerShellPolicyTests.PowerShellConfig_GetPowerShellPolicies_BothConfigFilesNotEmpty() in D:\a\1\s\test\xUnit\csharp\test_PSConfiguration.cs:line 383
```
PowerShellConfig_GetPowerShellPolicies_EmptyUserConfig
```powershell
Assert.NotNull() Failure
   at PSTests.Sequential.PowerShellPolicyTests.PowerShellConfig_GetPowerShellPolicies_EmptyUserConfig() in D:\a\1\s\test\xUnit\csharp\test_PSConfiguration.cs:line 397
```